src/Toml/Printer.hs

```diff
@@ -88,12 +88,21 @@ prettyKeyValue i = Text.concat . map kvText . HashMap.toList
     showText :: Show a => a -> Text
     showText = Text.pack . show
 
+    showDouble :: Double -> Text
+    showDouble d | isInfinite d && isNegate d = "-inf"
```

Comment:
Don't need to have `isNegate` function. You can just check `d < 0`. This should be enough.